nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_kernel.py

```diff
@@ -496,7 +496,7 @@ def build(self, target_model: NNCFNetwork) -> ElasticKernelHandler:
                 if max_kernel_size >= 3:
                     crop_op = ElasticKernelInputForExternalPadding(elastic_kernel_op, max_kernel_size)
                     op = UpdateInputs(crop_op).to(device)
-                    nncf_logger.warning('Padded input will be cropped for {}'.format(node_name))
+                    nncf_logger.warning(f'Padded input will be cropped for {node_name}')
```

Comment:
Propose moving to debug
@jpablomch please confirm that it won't hurt user experience

Reply:
@ljaljushkin Confirmed. Thanks!

Reply:
Done